src/workflow/default_executor.ts

```diff
@@ -54,3 +69,5 @@ export class DefaultWorkflowExecutor implements WorkflowExecutor {
     );
   }
 }
+
+export const chainExecutor = new DefaultWorkflowExecutor({ chainOutputs: true });
```

Comment:
Why are we exporting this default instance? It's not used anywhere.

Reply:
it is used in test cases